src/smartHandler.ts

```diff
@@ -143,6 +145,26 @@ export class SMARTHandler implements Authorization {
         }
     }
 
+    async getSearchFilterBasedOnIdentity(request: GetSearchFilterBasedOnIdentityRequest): Promise<SearchFilter[]> {
+        const fhirUser = this.getFhirUser(request.userIdentity);
+        const { hostname, resourceType, id } = fhirUser;
+

```

Comment:
Add minor comment something like:
```suggestion

// Create a SearchFilter to limit access to only resources that are referenced to the requesting user
```

Reply:
updated